src/main/java/com/codeborne/selenide/SelenideDriver.java

```diff
@@ -300,34 +336,41 @@ public boolean atBottom() {
     return executeJavaScript("return window.pageYOffset + window.innerHeight >= document.body.scrollHeight");
   }
 
+  @Nonnull
   public SelenideTargetLocator switchTo() {
     return driver().switchTo();
   }
 
   @CheckReturnValue
+  @Nullable
```

Comment:
why is it nullable? Don't see in selenium client javadoc any info https://www.javadoc.io/doc/org.seleniumhq.selenium/selenium-api/3.141.59/org/openqa/selenium/WebDriver.html#getCurrentUrl--

also in w3c spec there is no info about return null
https://www.w3.org/TR/webdriver/#get-current-url